server/routes/visitors.test.ts

```diff
@@ -279,125 +286,318 @@ describe('POST /visit/select-visitors/A1234BC', () => {
   })
 })
 
-describe('GET /visit/select-main-contact/A1234BC', () => {
-  it('should show an error if invalid prisoner number supplied', () => {
-    const req = request(app).get('/visit/select-main-contact/123')
+describe('GET /visit/additional-support/:offenderNo', () => {
+  beforeEach(() => {
+    visitSessionData = {
+      prisoner: {
+        name: 'prisoner name',
+        offenderNo: 'A1234BC',
+        dateOfBirth: '25 May 1988',
+        location: 'location place',
+      },
+    }
+  })
 
-    return req.expect('Content-Type', /html/).expect(res => {
-      expect(res.text).toContain('Invalid prisoner number supplied')
-    })
+  it('should render the additional support page with no options selected', () => {
+    sessionApp = appWithAllRoutes(null, null, null, null, systemToken, false, { visitSessionData } as SessionData)
+
+    return request(sessionApp)
+      .get('/visit/additional-support/A1234BC')
+      .expect(200)
+      .expect('Content-Type', /html/)
+      .expect(res => {
+        const $ = cheerio.load(res.text)
```

Comment:
This looks a great way to do this 🙂

Reply:
Feel I should declare creative inspiration from `hmpps-welcom-people-into-prison-ui` 😉!